### DISCUSSION TITLE
请问岛播放器自定义配置中可以正常使用osd-dimensions参数吗

### DISCUSSION THREAD

**Discussion by Ptilopsis01** [Richasy/Bili.Copilot#822] [Q&A]
如题，更新日志中说osd相关配置会无法生效，我想知道我在profile的判断中使用了osd-dimensions下的参数，请问能正常获取吗

**Comment by Richasy** [comment]
不能保证。

在哔哩助理中，新建了一个 win32 窗口用作 mpv 的渲染窗口，但是该窗口不接受任何鼠标或键盘消息，这意味着它仅仅是用来渲染视频画面的 `画布`，所以我会在更新日志里强调关于窗口和OSD的参数不能生效，因为无法观测。

也许你可以获取 osd-dimensions 参数的值，但并不保证准确。
